src/lib/TopLevel.hs

```diff
@@ -693,17 +696,17 @@ evalLLVM :: (Topper m, Mut n) => IxDestBlock n -> m n (Atom n)
 evalLLVM block = do
   backend <- backendName <$> getConfig
   logger  <- getFilteredLogger
-  (blockAbs, ptrVals) <- abstractPtrLiterals block
   let (cc, _needsSync) = case backend of LLVMCUDA -> (EntryFun CUDARequired   , True )
                                          _        -> (EntryFun CUDANotRequired, False)
   ImpFunctionWithRecon impFun reconAtom <- checkPass ImpPass $
-                                             toImpFunction backend cc blockAbs
+    toImpFunction backend cc $ Abs Empty block
```

Comment:
Perhaps remove the `Abs` too, since it's always empty?

Reply:
Yes, good point. I had thought that this was used in the standalone function path too but I was wrong.